apis/v1alpha2/httproute_types.go

```diff
@@ -455,16 +453,15 @@ type HTTPRouteMatch struct {
 	Method *HTTPMethod `json:"method,omitempty"`
 }
 
-// HTTPRouteFilter defines additional processing steps that must be completed
-// during the request or response lifecycle. HTTPRouteFilters are meant as an
-// extension point to express additional processing that may be done in Gateway
-// implementations. Some examples include request or response modification,
-// implementing authentication strategies, rate-limiting, and traffic shaping.
-// API guarantee/conformance is defined based on the type of the filter.
-// TODO(hbagdi): re-render CRDs once controller-tools supports union tags:
-// - https://github.com/kubernetes-sigs/controller-tools/pull/298
-// - https://github.com/kubernetes-sigs/controller-tools/issues/461
-// +union
+// HTTPRouteFilter defines processing steps that must be completed during the
+// request or response lifecycle. HTTPRouteFilters are meant as an extension
+// point to express processing that may be done in Gateway implementations. Some
+// examples include request or response modification, implementing
+// authentication strategies, rate-limiting, and traffic shaping. API
+// guarantee/conformance is defined based on the type of the filter.
+// TODO(hbagdi): re-render CRDs once controller-tools supports union tags: -
+// https://github.com/kubernetes-sigs/controller-tools/pull/298 -
+// https://github.com/kubernetes-sigs/controller-tools/issues/461 +union
```

Comment:
The `+union` should be on a line by itself. Separate TODO into a separate paragraph, or drop it into an issue.

Reply:
Good call, removed entirely: https://github.com/kubernetes-sigs/gateway-api/issues/854.